src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```

Comment:
you can remove all of this placeholder text, logos, and links until your left with either a blank screen, or a small placeholder like <div>React o-shop</div>.

Reply:
removed placeholder content